crates/bevy_tasks/src/task_pool.rs

```diff
@@ -239,6 +241,26 @@ impl TaskPool {
         Task::new(self.executor.spawn(future))
     }
 
+    /// Spawns a static future onto the thread pool. The returned `PollableTask` is not a future,
+    /// but can be polled in system functions on every frame update without being blocked on
+    pub fn spawn_pollable<T>(
+        &self,
+        future: impl Future<Output = T> + Send + 'static,
+    ) -> PollableTask<T>
+    where
+        T: Send + Sync + 'static,
+    {
+        let (sender, receiver) = bounded(1);
+        let task = self.spawn(async move {
+            let result = future.await;
+            match sender.send(result).await {
+                Ok(_) => {}
+                Err(_) => warn!("Could not send result of task to receiver"),
```

Comment:
This error message could/should be expanded. E.g.
```suggestion
                Err(_) => error!("Sending result for future {future_name} (`Future<Output={return_name}>`) failed, because the recieving `PollableTask` was dropped"),
```
where `future_name` is `std::any::type_name::<[F]>()`, i.e. the type name of the future and `return_name` was `std::any::type_name::<T>()`

Additionally, it might be worth tracking the caller location of this function for this diagnostic. That would need to be outside of the `async move` block.

Reply:
How to do the tracking of the caller location?

Reply:
The caller location, or caller id could be given optionally to the spawn interface to let the user decide what identification system to use. 

However I think this suggestion might be a better for for a later PR, it would be best introduced if
a. there's a real need for it from the users
b. later on when the initial feature is already introduced